src/components/CloneTable.tsx

```diff
@@ -11,7 +11,7 @@ interface CloneTableProps {
 const CloneTable: React.FC<CloneTableProps> = ({ dataSource }) => {
     // NOTE: once the clone data is filled in, we can use the id as the key
```

Comment:
Just checking - does `id` refer to `cell_line_id`(i.e.`cellLine.frontmatter.cell_line_id`)?